test/core.jl

```diff
@@ -223,4 +223,25 @@ Nabla.∇(::typeof(quad), ::Type{Arg{2}}, p, Y, Ȳ, A::Matrix, B::Matrix) = A*B
     @test oldvals[2:end] ≈ newvals[2:end]
 end
 
+@testset "rrules can deal with ZeroTangent()" begin
+    myfunc(i, j) = i + j
+
+    function ChainRulesCore.rrule(::typeof(myfunc), i, j)
+        y = myfunc(i, j)
+        # the pullback can't deal with ZeroTangent() (make sure Nabla deals with this)
+        myfunc_pullback(dy::Number) = NoTangent(), dy, dy
+        return y, myfunc_pullback
+    end
+
+    Nabla.generate_overload(Tuple{typeof(myfunc), Any, Any})
```

Comment:
Do we need to add a generic manually triggered refresh?
Does 
`ChainRulesOverloadGeneration.refresh_rules()` work? 
I think it should, it does have tests. (might not play so well with precompilation though, that i haven't tested)
https://juliadiff.org/ChainRulesOverloadGeneration.jl/stable/api.html#ChainRulesOverloadGeneration.refresh_rules-Tuple{}

Reply:
Tried it, but that doesn't seem to work. I will investigate more closely tomorrow, we are not in a rush for this anyway